### PR TITLE
#147180987 Build dashboard for unresolved stuff

### PR DESCRIPTION
**Pull request by EugeneBad**
#### What does this PR do?
Provide a a separate dashboard view for the user to access their failed jobs.

#### Description of Task to be completed?
GIVEN that a user has been notified of a failed job.
WHEN the user logs into the dashboard.
THEN they should have a view listing all their failed jobs which they have been notified about.
Heroku deployment: http://health-checks.herokuapp.com/

#### How should this be manually tested?
Contact me via slack '@eugenius' to get login credentials.

#### What are the relevant pivotal tracker stories?
#147180987

### CODE REVIEW

**Comment by NdagiStanley** [comment]
Let's fix these conflicts @EugeneBad

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,9 @@ __pycache__/
 .coverage
 hc.sqlite
```
Comment (NdagiStanley) [inline .gitignore]: Update this to be `hc.sqlite3`

Comment (NdagiStanley) [reply .gitignore]: Hence we'll need to remove the binary file: `hc.sqlite3` from version control

File: hc/front/views.py [hunk 2]
```diff
@@ -60,6 +63,68 @@ def my_checks(request):
     return render(request, "front/my_checks.html", ctx)
 
 
+@login_required
+def failed_checks(request):
```
Comment (NdagiStanley) [inline hc/front/views.py:67]: Good use of the decorator 💯

File: requirements.txt [hunk 3]
```diff
@@ -4,6 +4,8 @@ Django==1.10
 django_compressor==2.1
 djmail==0.11.0
 futures==3.0.3
+gunicorn
```
Comment (NdagiStanley) [inline requirements.txt]: I would advise that we have the versions of these packages: `gunicorn` and `whitenoise` explicitly stated.

**Review by NdagiStanley — CHANGES_REQUESTED** [review]
It is very interesting that you implemented desktop and mobile pages 💯 
I'll check the deployed app for what you've implemented.

File: hc/front/views.py (hunk 2 above)
Comment (EugeneBad) [reply hc/front/views.py:67]: Well i would have preferred the LoginRequiredMixin; i guess the decorator is okay for simplicity's sake!

**Pull request merged**